bids-validator/utils/files/index.js

```diff
@@ -4,6 +4,7 @@ import FileAPI from './FileAPI'
 import newFile from './newFile'
 import readFile from './readFile'
 import readOMEFile from './readOMEFile'
+import readBuffer from './readBuffer'
```

Comment:
I have a "general" question about `readOMEfile` and `readBuffer`. 
Are they used to read only the file header or do they read all the data as well? (If they read all the data, that could be problematic with large datasets and very big files for validation time).

Reply:
We need `readBuffer` because we need a buffer which is an array of bytes to represent the whole file and check if it's a tiff or bigtiff file based on the file signature. So, yes, `readBuffer` has to read all the data. But I think `readOMEfile` only reads the file header.

Reply:
Also, in ExifReader's source code, before it load's the metadata of image files, it loads the given file as a Buffer first as well. I think Buffer reading is unavoidable in the whole validation process.

Reply:
Ok, that makes sense, I understand that reading the data at least once is inevitable.
As long as we read the data only once, it's good for me, thanks for checking!